Tests/StencilSwiftKitTests/XCTest+Assertions.swift

```diff
@@ -0,0 +1,12 @@
+//
+// StencilSwiftKit
+// Copyright (c) 2017 SwiftGen
+// MIT Licence
+//
+
+import XCTest
+
+public func XCTAssertThrows(_ expression: @autoclosure () throws -> Any?, _ message: @autoclosure () -> String = "",
```

Comment:
Wow I don't even remember why I ended up writing it. I'll remove it and just use `XCTAssertThrowsError` instead.